code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -16,7 +16,7 @@
 	return
 
 /obj/item/weapon/gun/projectile/automatic/pistol/m1911
-	name = "\improper M1911 pistol"
+	name = "M1911"

```

Comment:
It has the pistol in hand sprite. It makes a loud bang noise. It sends bullets at you. What is confusing about that?